boneio/example_config/output32x5A.yaml

```diff
@@ -1,9 +1,9 @@
 # mcp1 - upper left 8 relays
 - id: OUT_01
-  kind: mcp
-  mcp_id: mcp1
-  pin: 0
-  output_type: light
+ kind: mcp
+ mcp_id: mcp1
+ pin: 0
+ output_type: light
```

Comment:
<img width="605" alt="image" src="https://user-images.githubusercontent.com/142751/170857751-cac3bc83-2f2a-45c1-a7f9-e75638d54ae3.png">

@maciejkrasuski ^^ I think this change causes issues, IMHO the previous version was OK